### ISSUE TITLE
freva-web: solr has lost "experiment_prefix" utility-> it affects web decadal analysis

### ISSUE THREAD

**Issue by eelucio** [FREVA-CLINT/freva-web#56]
As far as I am aware of, the [freva-web has lost](https://github.com/FREVA-CLINT/freva-web/blob/v2023.08.23/solr/views.py#L107) the solr/view.py `experiment_prefix` part:

```python
        elif facets == "experiment_prefix":
            args["experiment"] = args.pop("experiment_prefix")
            results = reorder_results(freva.count_values(facet="experiment", **args))
            results["experiment_prefix"] = remove_year(results.pop("experiment"))
        else:
            if "experiment_prefix" in args:
                args["experiment"] = args.pop("experiment_prefix")[0] + "*"
            results = reorder_results(freva.count_values(facet=facets, **args))
```

as it now uses [freva-databrowser endpoint instead](https://github.com/FREVA-CLINT/freva-web/blob/v2405.0.0/solr/views.py) that it does[ not consider it anymore (?)](https://github.com/FREVA-CLINT/freva-nextgen) and has broken the web query utility.

Basically any web query relying on that will not work anymore: that means that e.g. decadal data plugin web runs are broken.

this issue is related to

- [this issue wiht problems](https://trello.com/c/SK6nhBSr/405-problems-solr-facet-search-with-prefixes-unresponsive-in-web)

- [and will affect this general issue](https://trello.com/c/I7Lw9TRn/372-how-to-handle-cmip6-decadal-data-in-freva-mapping)

**Comment by antarcticrainforest** [comment]
I am not sure I entirely understand this, but this sounds to me that it's a plugin issue.

**Comment by eelucio** [comment]
I am afraid is a problem on the freva-web side, 

Some plugins (mostly decadal) rely on `experiment_prefix` that was part of the freva-web in order to search all decadal experiments [such as](https://www.xces.dkrz.de/databrowser/?project=dcpp&start=0):

```
> freva databrowser project=dcpp --facet experiment

dcppa-hindcast2005
dcppa-hindcast2006
dcppa-hindcast2007
dcppa-hindcast2008
dcppa-hindcast2009
dcppa-hindcast2010
...
```
where the prefix here would be `dcppa-hindcast`
right now the the web is unable to resolve that (it was before, but since the recent update where the piece of code I mentioned above was removed is not anymore)
